test/extended/openstack/utils.go

```diff
@@ -0,0 +1,94 @@
+package openstack
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"strconv"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	e2epod "k8s.io/kubernetes/test/e2e/framework/pod"
+	imageutils "k8s.io/kubernetes/test/utils/image"
+
+	o "github.com/onsi/gomega"
+)
+
+type KuryrNetwork struct {
+	Status struct {
+		SubnetID string `json:"subnetId"`
+	} `json:"status"`
+}
+
+func ElementExists(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+	return false
+}
+
+func RandomSuffix() string {
+	return strconv.Itoa(rand.Intn(10000))
+}
+
+func GetKuryrNetwork(clientSet *kubernetes.Clientset, namespace string) KuryrNetwork {
+	//TODO(itzikb): Replace a direct call to the API by extending the ClientSet
+	knetworkPath := fmt.Sprintf("/apis/openstack.org/v1/namespaces/%s/kuryrnetworks/%s", namespace, namespace)
+	data, _ := clientSet.RESTClient().
```

Comment:
is the parameter that is ignored a possible error? If yes, let's handle and if it's 404 code we can improve the check at line 53.

Reply:
Moving it now up the stack